stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -280,6 +287,9 @@ public class MessageListViewModel(
                                 messageItems = emptyList(),
                             )
                         is io.getstream.chat.android.offline.plugin.state.channel.MessagesState.Result -> {
+
+                            val scrollingState = shouldScrollToBottom(channelState.endOfNewerMessages.value)
```

Comment:
I'd rename this: getScrollingState() or something - shouldScrollToBottom is a name if we'd expect a boolean - true or false, not a complex data type